test/specs/tests.with.po.js

```diff
@@ -0,0 +1,54 @@
+const { expect } = require('chai');
+const { pages } = require('../pages/Pages');
+
+describe('Hometask test suite with PO', () => {
+    it('Perform sorting on the Inventory page', async () => {
+        await pages.loginPage.navigate();
+        await pages.loginPage.performLogin('standard_user', 'secret_sauce');
+        await pages.inventoryPage.sortItems();
```

Comment:
It's not obvious what sorting is applied. Could you please make method to perform some specific sorting defined within the test